.circleci/config.yml

```diff
@@ -35,16 +35,14 @@ jobs:
           command: PATH="venv/bin:$PATH" yamllint -c .yamllint.yaml .
       - run:
           name: PyTest with linters
-          # yamllint disable rule:line-length
+          # integration and java tests are run in a separate `integration` step;
+          # SQL and routine tests are split out into a separate `test-sql` test
+          # since those tests take the longest to run and running those tests
+          # in parallel speeds up CI
           command: |
-            if [ -n "$CIRCLE_PR_NUMBER" ]; then
-              echo "Cannot pass creds to forked PRs," \
-                "so skipping routine and SQL tests"
-              PATH="venv/bin:$PATH" script/entrypoint -m "not (routine or sql or integration or java)"
-            else
-              PATH="venv/bin:$PATH" script/entrypoint
-            fi
-          # yamllint enable rule:line-length
+            PATH="venv/bin:$PATH" script/entrypoint --black --flake8 \
+              --mypy-ignore-missing-imports --pydocstyle \
+              -m "not (routine or sql or integration or java)"
```

Comment:
`build` is one of the longest CI tasks. And while [reducing --numprocesses](https://github.com/mozilla/bigquery-etl/pull/2479/files#r743990341) improved the runtime (7m vs 9m), splitting out SQL tests (which run the slowest) makes things even faster.

Before `build` took ~9m
With this `build` is ~2m and `test-sql` ~5m